src/image/loading_displaying.js

```diff
@@ -238,19 +238,20 @@ p5.prototype.saveGif = async function(
   if (typeof duration !== 'number') {
     throw TypeError('Duration parameter must be a number');
   }
+
+  // extract variables for more comfortable use
+  const delay = (options && options.delay) || 0;  // in seconds
+  const units = (options && options.units) || 'seconds';  // either 'seconds' or 'frames'
+
   // if arguments in the options object are not correct, cancel operation
-  if (typeof options.delay !== 'number') {
+  if (typeof delay !== 'number' && delay !== null) {
```

Comment:
Looks good! Just for my understanding, was there a case that wasn't being handled that the `&& delay !== null` is needed for?